src/open-response/components/runtime.tsx

```diff
@@ -41,17 +30,6 @@ export const Runtime: React.FC<IProps> = ({ authoredState, interactiveState, set
           placeholder={authoredState.defaultAnswer || "Type answer here"}
         />
       </div>
-      {
-        authoredState.extraInstructions &&
-        <div className={css.extraInstructions}>{ authoredState.extraInstructions }</div>
-      }
-      {
-        !report &&
-        <div>
-          { submitButton }
-          { lockedInfo }
```

Comment:
We might need to do something more advanced here so the individual questions can control where the submitButton and lockedInfo go in their UI. I'm assuming that they get automatically added below the question with this change. I'm confident that we can figure out how to do that if and when it is needed.

Reply:
Right. To make it work with ScaffoldedQuestion, I thought I'd just let interactive provide a set of buttons that will be added to the bottom row + let it specify CSS class name (so it can be moved to the right).

If things get more complex, I think there are two approaches:
  - Interactive doesn't have to use this BaseQuestionApp. It can implement its own, it's not super complicated. I think we can a few different BaseApps if a few interactives share common patterns rather than trying to make one super configurable helper.
  - Hooks cannot be conditional (rules of hooks), but we can pass something like `isEnabled` to each of them. That way, we could turn on and off various features. So, an interactive could turn off `useRequiredQuestion` and then use this hook itself in the runtime (well, in fact we don't even have to turn it off, it won't hurt if it's in both places). This will work when we `useLARAInteractiveAPI` uses global iframe-phone and can be requested multiple times. Then, all the hooks will also become much simpler, as each hook will use `useLARAInteractiveAPI`.

Reply:
Yup sounds good.